dashboard/src/components/Tabs/Tests/ConfigsList.tsx

```diff
@@ -20,12 +21,34 @@ const ConfigsList = ({
   title,
   diffFilter,
 }: IConfigList): JSX.Element => {
+  const sortedConfigStatusCounts = useMemo(
```

Comment:
I suggest adding some short JSDoc on these like

```suggestion
  /**
   * Sorted with UNKNOWN_STRING last, then by descending error count, then by arch text alphabetically.
   */
  const sortedConfigStatusCounts = useMemo(
```

and what if we centralized them at `src/utils/utils.ts` or `src/utils/misc.tsx`, then we could use it with something like

```js
summaryBody.sort((a,b) => sortTextAndErrors(a.arch, b.arch))
configs.sort(sortTextAndErrors)
Object.entries(configStatusCounts).sort((a,b) => sortTextAndErrors({text: a[0], errors: a[1].FAIL}, {text: b[0], errors: b[1].FAIL}))
archCompilerErrors.sort((a, b) => sortTextAndErrors({text: a.arch, errors: a.status.FAIL}, {text: b.arch, errors: b.status.FAIL}))
```